services/wfm/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/IslHandler.java

```diff
@@ -94,16 +108,23 @@ private void handleSpeakerInput(Tuple input) throws PipelineException {
         command.apply(this);
     }
 
+    private void handleSpeakerFlowWorkerInput(Tuple input) throws PipelineException {
```

Comment:
Is it possible to use one method instead of the three methods (`handleUniIslCommand`,` handleSpeakerInput` and `handleSpeakerFlowWorkerInput`) to avoid code duplication?

Reply:
it's been maden originally here, I believe for the further usage